### PR TITLE
Documentation, again, almost done

### PR DESCRIPTION
**Pull request by Tristramg**

### CODE REVIEW

File: src/objects.rs [hunk 1]
```diff
@@ -118,39 +152,54 @@ impl Serialize for RouteType {
     }
 }
 
+/// Describes if and how a traveller can board or disboard the vehicle. See https://gtfs.org/reference/static/#stop_timestxt `pickup_type` and `dropoff_type`
```
Comment (antoine-de) [inline src/objects.rs:155]: alight instead of disboard no?

File: src/objects.rs [hunk 2]
```diff
@@ -172,50 +221,61 @@ impl<'de> Deserialize<'de> for TimepointType {
     }
 }
 
+/// A calender describes on which days the vehicle runs. See https://gtfs.org/reference/static/#calendartxt
 #[derive(Debug, Deserialize, Serialize)]
 pub struct Calendar {
+    /// Unique technical (not for the traveller) of this calendar
```
Comment (antoine-de) [inline src/objects.rs:227]: Unique technical *id* (not for the traveller) of this calendar

File: src/objects.rs [hunk 3]
```diff
@@ -267,52 +332,74 @@ impl Calendar {
...
+    /// Date where the service will be added or deleted
     pub date: NaiveDate,
+    /// Is the service added or deleted
     pub exception_type: Exception,
 }
 
+/// A physical stop, station or area. See https://gtfs.org/reference/static/#stopstxt
 #[derive(Debug, Serialize, Deserialize, Clone, Default)]
 pub struct Stop {
+    /// Unique technical (not for the traveller) of the stop
```
Comment (antoine-de) [inline src/objects.rs:364]: same here, missing `id`

File: src/objects.rs [hunk 4]
```diff
@@ -517,19 +667,32 @@ impl fmt::Display for RawTrip {
     }
 }
 
+/// A Trip is a vehicle that follows a sequence of [StopTime] on certain days. See https://gtfs.org/reference/static/#tripstxt
 #[derive(Debug, Default)]
 pub struct Trip {
+    /// Unique technical (not for the traveller) identifier for the Trip
```
Comment (antoine-de) [inline src/objects.rs:673]: same here, missing id

**Review by antoine-de — COMMENTED** [review]
😍 😍 

if that's the last pull request, maybe we can bump the minor version in order to release this to crates.io and to docs.rs?

File: src/objects.rs (hunk 4 above)
Comment (Tristramg) [reply src/objects.rs:673]: no, here it was missing, it was after the parenthesis (but I moved it before)

**Comment by Tristramg** [comment]
There is one last thing I wanted to do: add some text about the general format, the extensions and some design issues.

**Comment by Tristramg** [comment]
ok, I added the introduction that I wanted, so I bumped the version. I think that the documentation can be considered as finished.

**Review by antoine-de — APPROVED** [review]

**Pull request merged**